packs/bestiary-effects/effect-aspers-elemental-aura.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Asper's Elemental Aura",
     "system": {
         "description": {
-            "value": "<p><strong>Air</strong> You gain resistance 10 to electricity damage and double your speed.</p>\n<p><strong>Fire</strong> You gain resistance 10 to fire damage. Any creature within 30 feet that receives fire damage receives an extra @Damage[2d10[persistent,fire]] damage.</p>\n<p><strong>Water</strong> You gain resistance 10 to cold damage and fast healing 20.</p>"
+            "value": "<p>You gain resistance 10 to damage of the chosen element (electricity for air, fire, or cold for water). If air, you double your speed. If fire, any creature within 30 feet of you who receives fire damage receives an extra 2d10 persistent fire damage. If water, you gain fast healing 20.</p>"
```

Comment:
This is your change from #16874, isn't it? Probably don't wanna have it here too

Reply:
Good catch!